bundles/org.openhab.ui/web/src/assets/i18n/theme-switcher/en.json

```diff
@@ -7,7 +7,10 @@
   "about.darkMode.dark": "Dark",
   "about.navigationBarsStyle": "Navigation bars style",
   "about.miscellaneous": "Miscellaneous",
-  "about.miscellaneous.home.navbar": "Simple navigation bar on home page",
```

Comment:
@ghys Do you know how I can remove the existing translations for this setting? 
I need to get rid of them because there is a new text. Does CrowdIn do that automatically or do I have to remove this string from CrowdIn and from the i18n assets?

Reply:
I have no idea but I can check.

Reply:
Thanks! You can merge as soon as you want to - I will be away from keyboard next week.